org.p2panda.aardvark.json

```diff
@@ -46,6 +46,9 @@
                     "url" : "./",
                     "branch" : "HEAD"
                 }
+            ],
+            "config-opts" : [
+                "--libdir=lib"
```

Comment:
what is this for?

Reply:
I dont know! Builder added that automatically 😐

Reply:
oh, it does the same for me... let's just go with it.